scripts/check_license

```diff
@@ -30,8 +30,7 @@ function filterExcludedFiles {
 		| grep -v "\.md$" \
 		| grep -v "\.pb\.go$" \
 		| grep -v ".gitignore" \
-		| grep -v "ci.properties" \
-        	| grep -v "\.xml$" \
+        | grep -v "\.xml$" \
```

Comment:
nit: looks like you used spaces instead of tabs; this stands out.

Reply:
It was already spaces, my change just drew attention to it, I'll run shfmt across the scripts directory, I'm sure it ugly